x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style.js

```diff
@@ -334,6 +335,42 @@ export class VectorStyle extends AbstractStyle {
     return fieldMeta ? fieldMeta : fieldMetaFromLocalFeatures;
   };
 
+  _getFieldFormatter = fieldName => {
+    const dynamicProps = this.getDynamicPropertiesArray();
+    const dynamicProp = dynamicProps.find(dynamicProp => {
+      return fieldName === dynamicProp.getField().getName();
+    });
```

Comment:
Could you extract this boilerplate to new helpermethod `_getDynamicPropertyByFieldName` and reuse here? https://github.com/elastic/kibana/blob/1adadcd18b6029a61404ccd4126d4367bec44067/x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style.js#L301-L304